torch/distributed/fsdp/wrap.py

```diff
@@ -9,6 +9,14 @@
 import torch.nn as nn
 
 
+def always_wrap_policy(*args, **kwargs) -> bool:
+    """
+    A simple wrapper policy that always returns ``True``,
+    i.e. when passed as the `auto_wrap_policy` into FSDP,
+    this will result in all submodules being wrapped.
```

Comment:
nit: result in all submodules being wrapped, and each submodule is wrapped as a separated FSDP instance